docs/results_files.md

```
@@ -32,19 +32,17 @@ multiqc_DNA.html| html  | Aggregated QC values for entire sequence run, open in
 {sample}/cnv_sv/{sample}.manhattan.png | png | Manhattan plot for CNVpytor results |
 {sample}/cnv_sv/{sample}.manta_diploidSV.vcf.gz | vcf.gz  | Compressed VCF for SV calls called by Manta |
 {sample}/cnv_sv/{sample}.manta_diploidSV.vcf.gz.tbi | tbi | Index for the Manta VCF file |
-{sample}/cnv_sv/{sample}.tiddit.vcf.gz | vcf.gz | Compressed TIDDIT VCF file |
-{sample}/cnv_sv/{sample}.tiddit.vcf.gz.tbi | tbi | Index for the TIDDIT VCF file |
 {sample}/cnv_sv/{sample}.melt.vcf.gz | vcf.gz | Compressed MELT VCF file containing mobile element insertions |
 {sample}/cnv_sv/{sample}.melt.vcf.gz.tbi | tbi | Index for the MELT VCF file |
 {sample}/cnv_sv/{sample}.scramble.vcf.gz | vcf.gz | Compressed Scramble VCF file containing mobile element insertions |
 {sample}/cnv_sv/{sample}.scramble.vcf.gz.tbi | tbi | Index for the Scramble VCF file |
-{sample}/cnv_sv/{sample}.svdb_merged.vcf.gz | vcf.gz | Compressed VCF containing the merge of SV and CNV calls from Manta, Tiddit, CNVpytor, MELT and Scramble |
+{sample}/cnv_sv/{sample}.svdb_merged.vcf.gz | vcf.gz | Compressed VCF containing the merge of SV and CNV calls from Manta, CNVpytor, MELT and Scramble |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: trailing_only; Missing leading pipe

(MD055, table-pipe-style)